plugins/commands/removeTag.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const boom = require('@hapi/boom');
+const joi = require('joi');
+const schema = require('screwdriver-data-schema');
+const baseSchema = schema.models.commandTag.base;
+/* Currently, only build scope is allowed to tag command due to security reasons.
+ * The same pipeline that publishes the commands has the permission to tag it.
+ */
+
+module.exports = () => ({
+    method: 'DELETE',
+    path: '/commands/{namespace}/{name}/tags/{tagName}',
+    options: {
+        description: 'Delete a command tag',
+        notes: 'Delete a specific commands',
+        tags: ['api', 'commands'],
+        auth: {
+            strategies: ['token'],
+            scope: ['build']
+        },
+        handler: async (request, h) => {
+            const { pipelineFactory, commandFactory, commandTagFactory } = request.server.app;
+            const { pipelineId, isPR } = request.auth.credentials;
+            const { namespace, name } = request.params;
+            const tag = request.params.tagName;
+
+            console.log(name);
+
+            return commandTagFactory
+                .get({ namespace, name, tag })
+                .then(commandTag => {
+                    if (!commandTag) {
+                        throw boom.notFound('Commands tag does not exist');
```

Comment:
How about little more detailed log?

```suggestion
                        throw boom.notFound(`Command tag ${tag} does not exist for ${namespace}/${name}`);
```

Reply:
thank you